ckanext/switzerland/dcat/profiles.py

```diff
@@ -293,6 +322,220 @@ def parse_dataset(self, dataset_dict, dataset_ref):  # noqa
+                if value:
+                    g.add((dataset_ref, DCT.title, Literal(value, lang=key)))
+
+        # LandingPage
+        g.add((dataset_ref, DCAT.landingPage,
+               Literal(dataset_dict['url'])))
+
+        # multilang Keywords
+        for lang, keywords in dataset_dict.get('keywords').iteritems():
+            if keywords:
+                for keyword in keywords:
+                    g.add((dataset_ref, DCAT.keyword, Literal(keyword, lang=lang)))  # noqa
+
+        # Dates
+        items = [
+            ('issued', DCT.issued, ['metadata_created'], Literal),
+            ('modified', DCT.modified, ['metadata_modified'], Literal),
+        ]
+        self._add_date_triples_from_dict(dataset_dict, dataset_ref, items)
+
+        # Update Interval
+        accrual_periodicity = dataset_dict.get('accrual_periodicity')
+        if accrual_periodicity:
+            g.add((
+                dataset_ref,
+                DCT.accrualPeriodicity,
+                URIRef(accrual_periodicity)
+            ))
+
+        # Lists
+        items = [
+            ('language', DCT.language, None, Literal),
+            ('theme', DCAT.theme, None, URIRef),
+            ('conforms_to', DCT.conformsTo, None, Literal),
+            ('alternate_identifier', ADMS.identifier, None, Literal),
+            ('documentation', FOAF.page, None, Literal),
+            ('related_resource', DCT.relation, None, Literal),
+            ('has_version', DCT.hasVersion, None, Literal),
+            ('is_version_of', DCT.isVersionOf, None, Literal),
+            ('source', DCT.source, None, Literal),
+            ('sample', ADMS.sample, None, Literal),
+        ]
+        self._add_list_triples_from_dict(dataset_dict, dataset_ref, items)
+
+        # Contact details
+        if any([
+            self._get_dataset_value(dataset_dict, 'contact_uri'),
+            self._get_dataset_value(dataset_dict, 'contact_name'),
+            self._get_dataset_value(dataset_dict, 'contact_email'),
+            self._get_dataset_value(dataset_dict, 'maintainer'),
+            self._get_dataset_value(dataset_dict, 'maintainer_email'),
+            self._get_dataset_value(dataset_dict, 'author'),
+            self._get_dataset_value(dataset_dict, 'author_email'),
+        ]):
+
+            contact_uri = self._get_dataset_value(dataset_dict, 'contact_uri')
+            if contact_uri:
+                contact_details = URIRef(contact_uri)
+            else:
+                contact_details = BNode()
+
+            g.add((contact_details, RDF.type, VCARD.Organization))
+            g.add((dataset_ref, DCAT.contactPoint, contact_details))
+
+            items = [
+                ('contact_name', VCARD.fn, ['maintainer', 'author'], Literal),
+            ]
+            maintainer_email = self._get_dataset_value(
+                dataset_dict,
+                'maintainer_email'
+            )
+            g.add((contact_details, VCARD.hasEmail, URIRef(maintainer_email)))
```

Comment:
dataset_dict['contact_points'] contains name and email, do not use author and maintainer directly